root/client/src/util/tablehelpers.js

```diff
@@ -26,18 +36,34 @@ export const classTopPosition = (minutes) => {
 }
 
 export const classHeight = (duration) => {
-  if (duration === '15') {
-    return 'h-25';
-  } else if (duration === '30') {
-    return 'h-50';
-  } else if (duration === '45') {
-    return 'h-75';
-  } else if (duration === '60') {
-    return 'h-100';
+  if (duration === 15) {
+    return 'h-15';
+  } else if (duration === 30) {
+    return 'h-30';
+  } else if (duration === 45) {
+    return 'h-45';
+  } else if (duration === 60) {
+    return 'h-60';
+  } else if (duration === 120) {
+    return 'h-120'
   }
```

Comment:
hm since these are all equal in structure, can probs do something like

```suggestion
  return `h-${duration}`;
```